drivers/nrf_802154_sl/Kconfig

```diff
@@ -0,0 +1,5 @@
+#
```

Comment:
If it's empty, then we should not add it.

Reply:
This is because it would greatly simplify our further modifications and release (PRs). Whole drivers/nrf_802154_sl directory will be constructed as a result of some release script. Adding just modifications to our internal drivers/nrf_802154_sl/Kconfig would be much simpler than also modification of outer drivers/Kconfig. But `rsource "nrf_802154_sl/Kconfig"` found in outer Kconfig requires this one to exist. It is not harmful.
@rlubos Do you still insist on removing this? This will make our release more difficult.

Reply:
First of all, I don't insist, just suggest. I just really don't understand how adding `rsource "nrf_802154_sl/Kconfig"`in a future release, when some Kconfig values are actually added, will make the process more difficult. But if you prefer it this way, I guess it could stay, unless other reviewer objects.

Reply:
I must agree with @rlubos I don't see how creating an empty file simplifies anything.

Furthermore it could confuse users.
If this is because you have a private side-repo for building, then the Zephyr module is exactly designed in a way you can source the Kconfig in that repo when it's present.

I see no reason to have empty files in nrf repo.